notebook/static/notebook/js/actions.js

```diff
@@ -211,21 +217,21 @@ define(function(require){
                 env.notebook.select(index);
             }
         },
-        'copy-selected-cell' : {
+        'copy-cell' : {
             icon: 'fa-copy',
             help_index : 'ef',
             handler : function (env) {
                 env.notebook.copy_cell();
             }
         },
-        'paste-cell-before' : {
+        'paste-cell-above' : {
```

Comment:
I'm -0.7 on above / below, i would prefer `previous/next` as this apply to non-vertical interface (like slides).